content/en/docs/tasks/debug-application-cluster/debug-application.md

```diff
@@ -142,10 +142,10 @@ kubectl get pods --selector=name=nginx,type=frontend
 to list pods that match this selector.  Verify that the list matches the Pods that you expect to provide your Service.
 
 If the list of pods matches expectations, but your endpoints are still empty, it's possible that you don't
-have the right ports exposed.  If your service has a `containerPort` specified, but the Pods that are
+have the right ports exposed.  If your service has a `targetPort` specified, but the Pods that are
 selected don't have that port listed, then they won't be added to the endpoints list.
 
-Verify that the pod's `containerPort` matches up with the Service's `targetPort`
+Verify that the pod's `targetPort` matches up with the Service's `targetPort`
 
 #### Network traffic is not forwarded
```

Comment:
The first paragraph here is wrong in general (or at least depends on the proxy impl).   It used to be true when kube-proxy ran in userspace, but that changed a long time ago.   If it connects and is dropped then it actually did reach SOMETHING.